### ISSUE TITLE
Label does not stick after view change

### ISSUE THREAD

**Issue by crescendochu** [ProjectSidewalk/SidewalkWebpage#3353]
##### Brief description of problem/feature
Maybe I am wrong but I thought the labels would still stick in place after gsv view change? Both of the participants mentioned this was confusing as they don't remember if they have labeled something already or not.
https://github.com/ProjectSidewalk/SidewalkWebpage/assets/60413478/84443ff0-4cf0-44a4-a5b0-0027a25ce675

**Comment by misaugstad** [comment]
Unfortunately we don't have a good way to show labels on different panos! It's been like this since the beginning :'( We show a popup sometime early in the first mission to try and explain this, and to suggest looking at the mini map to help remember where your labels are. Not something we can fix right now, unfortunately!

**Comment by jonfroehlich** [comment]
Right, to be clear, this is the way Project Sidewalk has worked (and all of its precursors) since the very beginning.

To get labels to stick to new, unlabeled panoramas, we would need to use computer vision to find the same visual features that were labeled in pano1 and propagate them to pano2. This has just not been previously possible with CV... though it could be now with modern CV.

On the other hand, I don't consider this a high priority issue to investigate since this is always the way Project Sidewalk has worked and it seems most people eventually "get it." 

But I suppose you could mention this during training. It's just not the kind of thing that seems to sink in (for example, you're just now realizing it after 1+ years with Project Sidewalk)--and you kind of have to explain it after they've used the tool a bit (because, again, it's not so straightforward to understand unless you've used the tool).

**Comment by misaugstad** [comment]
Reopening this issue, since we may have a solution hidden in Google's API, recently found by @jsomeara! We should try out the [fromLatLngToDivPixel](https://developers.google.com/maps/documentation/javascript/reference/overlay-view#MapCanvasProjection.fromLatLngToDivPixel) to get the location on the image for you labels, and see how that looks across images!

**Comment by jsomeara** [comment]
### Investigation on GSV Location Extraction Method vs. Regression Approximation

**Method:**

I randomly sampled 100 PS curb ramp labels and extracted their location using the regression method (old method) and the gsv method (new method). These extracted locations were overlayed onto aerial imagery. For each PS label, I manually classified which one visually appeared to be closer to the actual curb ramp (per the aerial imagery). If it was hard to tell where the actual curb ramp was (e.g., due to trees occlusion, shadows, low res, low contrast), I classified as "can't tell who wins."

**Results on the investigation:**

- old approximation method wins: 42 (42%)
- GSV-method wins: 37 (37%)
- can't tell who wins (e.g., due to tree occlusion): 21 (21%)
- Total: 100

Wow! That's definitely a surprising result for me. I thought for sure the new GSV-method would win. What I observed was that often, the PS users' labels would be misplaced ever so slightly that it would completely throw off the GSV-method, while the old approximation method is much more tolerant to users' impreciseness. I think the reason I thought the GSV-method would be so much better is because during testing, I was only really testing with points that I knew to be accurate.

I think this means we should revert back to the old method for converting marker -> lat lon, but we can still use the new method to convert lat lon -> marker, enabling sticky labels. But maybe it's better to use the regression method for lat lon -> marker as well? Not really sure.

https://github.com/ProjectSidewalk/gsv-location-extraction-analysis
